auth/src/main/java/org/jboss/pnc/auth/ExternalAuthFacade.java

```diff
@@ -52,7 +53,8 @@
      */
     public ExternalAuthFacade(InputStream keycloakConfiguration) throws Exception{
```

Comment:
Repeating the package name (auth) in class name is considered as anti-pattern. And abbreviations should be avoided in public API.